src/main/java/kitchenpos/domain/OrderTable.java

```diff
@@ -1,50 +1,105 @@
 package kitchenpos.domain;
 
+import javax.persistence.*;
+import java.util.List;
+import java.util.Objects;
+
+import static kitchenpos.constants.ErrorCodeType.GUEST_NOT_NULL_AND_ZERO;
+import static kitchenpos.constants.ErrorCodeType.TABLE_GROUP_NOT_NULL;
+
+@Entity
 public class OrderTable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Long tableGroupId;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "table_group_id")
+    private TableGroup tableGroup;
     private int numberOfGuests;
-    private boolean empty;
 
-    public OrderTable() {
+    @Column(name = "empty")
+    private boolean isEmpty;
+
+    protected OrderTable() {
     }
 
-    public OrderTable(Long id, Long tableGroupId, int numberOfGuests, boolean empty) {
+    public OrderTable(Long id, TableGroup tableGroup, int numberOfGuests, boolean isEmpty) {
         this.id = id;
-        this.tableGroupId = tableGroupId;
+        this.tableGroup = tableGroup;
         this.numberOfGuests = numberOfGuests;
-        this.empty = empty;
+        this.isEmpty = isEmpty;
     }
 
-    public Long getId() {
-        return id;
+    public OrderTable(TableGroup tableGroup, int numberOfGuests, boolean isEmpty) {
+        this.tableGroup = tableGroup;
+        this.numberOfGuests = numberOfGuests;
+        this.isEmpty = isEmpty;
     }
 
-    public void setId(final Long id) {
-        this.id = id;
+    public OrderTable(int numberOfGuests, boolean isEmpty) {
+        this.numberOfGuests = numberOfGuests;
+        this.isEmpty = isEmpty;
     }
 
-    public Long getTableGroupId() {
-        return tableGroupId;
+    public void ungroup() {
+        this.tableGroup = null;
     }
 
-    public void setTableGroupId(final Long tableGroupId) {
-        this.tableGroupId = tableGroupId;
+    public void changeEmpty(boolean isEmpty, List<Order> order) {
+        validCheckIsNotNullTableGroup();
+        validCheckIsOrderStatusInCookingAndMeal(order);
+        this.isEmpty = isEmpty;
```

Comment:
```suggestion
    public void changeEmpty(boolean isEmpty) {
        validCheckIsNotNullTableGroup();
        this.isEmpty = isEmpty;
```
`OrderTable`은 `Order`를 갖고있지 않은데 `order`를 검증하는게 어색하네요. 🤔